internal/resolution/manifest/maven.go

```diff
@@ -301,6 +309,28 @@ func (m MavenManifestIO) mergeParents(ctx context.Context, result *maven.Project
 	return result.Interpolate()
 }
 
+// Maven looks for the parent POM first in 'relativePath',
+// then the local repository '../pom.xml',
+// and lastly in the remote repo.
+func MavenParentPOMPath(currentPath, relativePath string) string {
+	if relativePath == "" {
+		relativePath = "../pom.xml"
+	}
+	path := filepath.Join(filepath.Dir(currentPath), relativePath)
+	if info, err := os.Stat(path); err == nil {
+		if !info.IsDir() {
+			return path
+		}
```

Comment:
Wondering what maven does if the `relativePath` points to a file that is not named `pom.xml`

Reply:
this is acceptable if the file exists and is a valid parent pom.xml, otherwise, maven tries to download from upstream.